packages/x-components/src/__stubs__/index.ts

```diff
@@ -0,0 +1,12 @@
+export * from './banners-stubs.factory';
```

Comment:
We still need to update all the imports that use functions from the `__stubs__` folder so they use this new index file. We can make it here or in a new task (my vote is for a new task)

Reply:
Yes I think so.

Reply:
I'll create the task.

Reply:
https://searchbroker.atlassian.net/browse/EX-4590